.changelog/3234.feature.md

```diff
@@ -0,0 +1,6 @@
+go/control: Add per-runtime status report
+
+An additional `runtimes` field has been added to the output of the
```

Comment:
Might be nice to also update the example command docs, with a runtime node example https://github.com/oasisprotocol/oasis-core/blob/master/docs/oasis-node/cli.md

Reply:
Done.